vscode-test-runner/workspace-1/.joyride/src/integration_test/runner.cljs

```diff
@@ -3,31 +3,34 @@
             [integration-test.db :as db]
             [promesa.core :as p]))
 
+(defn- write [& xs]
+  (apply js/process.stdout.write xs))
```

Comment:
This won't work as write only expects a single string. I think you need to call write multiple times for each x in xs or make a string yourself and then feed it to write.

Reply:
Yeah ... of course! 😄